### PR TITLE
Update event registration email text

### LINKED ISSUE #1050: Add text for email users receive after registering for an event
Subject: You are registered for [EVENT NAME] on [EVENT DAY AND DATE]
From: noreply@treescount.nycgovparks.org
To: helpful@mapper.com

Thank you for signing up for a TreesCount! 2015 event!  

We have received your RSVP to [EVENT NAME], hosted by [GROUP NAME]. Here are the event details:

When: [EVENT DATE – EVENT TIME]
Where: [LOCATION ADDRESS AND LOCATION DESCRIPTION]
Contact: [CONTACT NAME – CONTACT EMAIL]

You can view the full details of the event, cancel your reservation, or check-in when the event starts by following this link: [EVENT DETAILS URL]

### PR DESCRIPTION
**Pull request by kdeloach**
Fixes #1050

### CODE REVIEW

**Comment by kdeloach** [comment]
@azavea-bot rebuild

**Comment by maurizi** [comment]
+1

**Comment by maurizi** [comment]
@azavea-bot rebuild

File: src/nyc_trees/apps/mail/libs.py [hunk 1]
```diff
@@ -54,7 +54,6 @@ def send_to(user, message_type, *args, **kwargs):
         'from_email': from_email,
         'subject': subject,
         'body_text': body_text,
-        'sent_at': now()
```
Comment (hectcastro) [inline src/nyc_trees/apps/mail/libs.py]: This is still imported but not used.

Comment (kdeloach) [inline src/nyc_trees/apps/mail/libs.py]: Thanks... been kind of ignoring Jenkins failed/unstable build status since there's been so many lately.

**Comment by maurizi** [comment]
@azavea-bot rebuild

**Pull request merged**